jwql/website/apps/jwql/monitor_models/bad_pixel.py

```diff
@@ -210,11 +211,11 @@ class Meta:
         app_label = 'monitors'
 
 
-class NirspecBadPixelStats(models.Model):
+class NIRSpecBadPixelStats(models.Model):
     entry_date = models.DateTimeField(unique=True)
     detector = models.CharField(blank=True, null=True)
-    x_coord = models.TextField(blank=True, null=True)  # This field type is a guess.
-    y_coord = models.TextField(blank=True, null=True)  # This field type is a guess.
+    x_coord = ArrayField(models.IntegerField())
```

Comment:
Much better than guessing, thank you for updating this.

Reply:
Not to mention that the guess doesn't work (by which I mean if PostgreSQL has a field that's an array of integers, and you give it a string '[1, 2]', it doesn't know what to do with it, but it's pretty sure it's not an array of integers, so it will refuse to do the insert.

Reply:
Yeah, that's also a very valid point 😅 thank you!